adaptive_ai/adaptive_ai/core/config.py

```
@@ -1,90 +1,264 @@
+    description: str = "Intelligent LLM Infrastructure with Smart Model Selection"
+    environment: str = "development"
+    debug: bool = False
+
+
+class ServerConfig(BaseModel):
+    """Server configuration."""
 
-class Settings(BaseSettings):
-    # Server settings
-    port: int = 8000
     host: str = "0.0.0.0"
```

Ruff (0.11.9)

Possible binding to all interfaces

(S104)